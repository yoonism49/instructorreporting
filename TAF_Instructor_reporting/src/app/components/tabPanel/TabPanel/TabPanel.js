// @flow weak

import React      from 'react';
import PropTypes  from 'prop-types';

const TabPanel = ({
  children
}) => (
  <section className="panel general">
    {children}
  </section>
);

TabPanel.propTypes = {
  children: PropTypes.node.isRequired
};

export default TabPanel;
