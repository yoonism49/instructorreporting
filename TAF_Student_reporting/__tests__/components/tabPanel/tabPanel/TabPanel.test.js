
'use strict';

import React              from 'react';
import TabPanel           from '../../../../src/app/components/tabPanel/tabPanel/TabPanel';
import renderer           from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme

describe('TabPanel component', () => {
  it('renders as expected', () => {
    const component = renderer.create(
      <div>
        <TabPanel>
          <p>TabPanel children here</p>
        </TabPanel>
      </div>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
