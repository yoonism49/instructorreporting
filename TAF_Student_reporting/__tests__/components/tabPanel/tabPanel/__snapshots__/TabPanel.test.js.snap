// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabPanel component renders as expected 1`] = `
<div>
  <section
    className="panel general"
  >
    <p>
      TabPanel children here
    </p>
  </section>
</div>
`;
